store.css

```diff
@@ -0,0 +1,243 @@
+/* --==================== general styles ====================-- */
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+h1, h2, .about-text{
+  font-weight: lighter;
+}
+
+a{
+  text-decoration: none;
+}
+
+ /* --==================== HEADER ====================-- */
+ .header_logo {
+    height: 40px;
+    float: left;
+    position: relative;
+    padding-left: 100px;
+    /*margin: 10px 15px 15px 10px;*/
+  }
+
+  #header-store{
+    background: rgba(196, 196, 196, 0.1);
+  }
+
+ /* --==================== NAVBAR ====================-- */
+ 
+ .nav_icon-text{
+  font-weight: 450;
+  color:#6b6b6b;
+  padding: 15px;
+}
+ 
+ .nav_icon{
+  font-weight: 450;
+  color:#9c9a9a;
+  padding: 15px;
+ }
+
+ .nav_icon:hover{
+   color: #d65924;
+
+ }
+
+ .container{
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 60px;
+  list-style: none;
+ }
+
+ /* --==================== FEATURE SECTION ====================-- */
+
+*{
+    margin: 0;
+    padding: 0;
+}
```

Comment:
Mske sure you keep any kind of generic CSS like this at the very top of your file (and limit it to the strict necessary!)